src/build.h

```diff
@@ -239,9 +245,9 @@ struct Builder {
 struct BuildStatus {
   explicit BuildStatus(const BuildConfig& config);
   void PlanHasTotalEdges(int total);
-  void BuildEdgeStarted(Edge* edge);
-  void BuildEdgeFinished(Edge* edge, bool success, const string& output,
-                         int* start_time, int* end_time);
+  void BuildEdgeStarted(Edge* edge, int64_t start_time_millis);
+  void BuildEdgeFinished(Edge* edge, int64_t end_time_millis, bool success,
+                         const string& output);
```

Comment:
In the spirit of https://github.com/ninja-build/ninja/pull/1614

Consider using `const std::string& output` instead.